lib/writeLog.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+var config = require('../config/secrets');
+var logger = require('./logger');
+
+var logFlag = config.get().logger.api.log_user_activity;
+
+var writer = {};
+
+writer.writeIncident = function(req, incident, action) {
+  if (logFlag === false) return {};
```

Comment:
That's now. Just before the latest commit, an object used to be passed back to the controller, which would then call logActivity on it. :P